bundles/org.openhab.binding.somneo/src/main/java/org/openhab/binding/somneo/internal/SomneoBindingConstants.java

```diff
@@ -0,0 +1,74 @@
+/**
+ * Copyright (c) 2010-2022 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.somneo.internal;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.openhab.core.thing.ThingTypeUID;
+
+/**
+ * The {@link SomneoBindingConstants} class defines common constants, which are
+ * used across the whole binding.
+ *
+ * @author Michael Myrcik - Initial contribution
+ */
+@NonNullByDefault
+public class SomneoBindingConstants {
+
+    private static final String BINDING_ID = "somneo";
+
+    // List of all Thing properties
+    public static final String PROPERTY_VENDOR_NAME = "Philips";
+
+    // List of all Thing Type UIDs
+    public static final ThingTypeUID THING_TYPE_HF367X = new ThingTypeUID(BINDING_ID, "hf367x");
```

Comment:
Personally, I only have the HF3671/01 device with which I could test it. The project "homebridge-somneo" writes that the model HF3670/60 is supported. I think that all HF367X models are supported, I extracted this type name from the API. Unfortunately I did not find an overview of the existing models. I would adjust the description so that all HF367X are supported.